docs/application/vstools/tools/dotnet-hotreload.md

```diff
@@ -0,0 +1,49 @@
+# Tizen .NET Hot Reload
+
+With Hot Reload you can now modify your apps managed source code while the application is being debugged, without the need to stop and restart the debugging.
+After starting the debugging of the application on the emulator or real device, you can apply the changed file without stopping the debugging of the application.
+With Hot Reload changes are applied only on the second entrance to the function. If the change you made is supported, your app will now be patched while it's running with your new logic. You should see the changes in your app’s behavior the next time the updated code is re-executed by either your action or by a timer triggering the code.
+
+## Enable Tizen .NET Hot Reload
+
+Go to **Tools** menu in Visual Studio, and select **Tools > Options...** from the dropdown menu. **Options** window will open.
+
+**1.** Go to **Tizen &gt; Tools** and check the **Enable Hot Reload** checkbox.
+
+**Figure: Tizen options page to enable .NET Hot Reload**
+
+![Tizen options page to enable .NET Hot Reload](media/enable_tizen_hotreload.png)
+
+**2.** Go to **Debugging &gt; .NET / C++ Hot Reload** and check all the checkboxes.
+
+**Figure: VS options page to enable .NET Hot Reload**
+
+![Tizen options page to enable .NET Hot Reload](media/enable_vs_hotreload.png)
+
+
+## Supported/Unsupported changes and architecture
+Currently, Tizen .NET Hot Reload is supported on aarch64 and armv7l.
+No matter how you use .NET Hot Reload please be aware that some changes are not supported at runtime and will prompt you with a rude edit dialog and require you to restart your app in order to apply the changes. [Supported Edits in Tizen .NET Hot Reload](https://github.com/dotnet/roslyn/blob/main/docs/wiki/EnC-Supported-Edits.md)
```

Comment:
External links in a .md file open in a new window
```suggestion
No matter how you use .NET Hot Reload please be aware that some changes are not supported at runtime and will prompt you with a rude edit dialog and require you to restart your app in order to apply the changes. [Supported Edits in Tizen .NET Hot Reload](https://github.com/dotnet/roslyn/blob/main/docs/wiki/EnC-Supported-Edits.md){:target="_blank"}
```

Reply:
@omar-srbd Hi, could you review it again ?

Reply:
@om90prakash, as per the commits history in this PR, I can see that this specific change was not made in previous commits.
The instance where the change was made and had to be reverted was in the video links of **lines 39 & 49**.
The change requested here is for an external link redirecting users to a different site and hence should open in a new tab.